### ISSUE TITLE
Section with pinned/followed roadmaps on "My Profile" page

### ISSUE THREAD

**Issue by grafanaKibana** [kamranahmedse/developer-roadmap#6968]
### Feature Description

As a user who wants to continuously progress with some roadmap, I want to have the ability to quickly find the roadmaps that I'm following and progressing. 

EDIT: Thanks to @fellalli who pointed out that they are appearing on the home page. But for me as a user, it is not so obvious. Before creating this ticked I spent 5 mins trying to find a list of my "Favorites" and gave up 🙁

I wish they also appeared on the "My Profile" page, with no started roadmaps included (Now they appear only if you finished at least one topic). Moreover, I have started some community roadmaps that one not appear neither on the home page or "Activity" section.

**Comment by fellalli** [comment]
They appear at the top of https://roadmap.sh/ when you’re logged in, don’t they?

**Comment by grafanaKibana** [comment]
They do! But from a UX perspective, it is not so obvious that my pinned roadmaps will be on the project's main page. I did not find them before you mentioned they were there. I wish they also appeared on the "My Profile" page, with no started included (Now they appear only if you finished at least one topic). Moreover, I have started some community roadmaps that one not appear neither on the home page or "Activity" section. I will modify the Feature Description to be more precise In my desires 🙂
